src/DDTrace/Integrations/Lumen/LumenSandboxedIntegration.php

```diff
@@ -68,9 +68,11 @@ function (SpanData $span, $args) use ($rootSpan, $integration, $appName) {
             'Laravel\Lumen\Application',
             'handleFoundRoute',
             [
-                'prehook' => function (SpanData $span, $args) use ($rootSpan) {
+                // convert to non-tracing API
+                'posthook' => function (SpanData $span, $args) use ($rootSpan) {
```

Comment:
Not sure about the scope of this change. Isn't prehook returning `false` supported on 5.6 with sandboxing api, at this stage?

Reply:
If not, should we change also `handleUncaughtException` and `sendExceptionToHandler`?

Reply:
> Isn't prehook returning false supported on 5.6 with sandboxing api, at this stage?

No.

> If not, should we change also handleUncaughtException and sendExceptionToHandler?

Probably yes; looking into it.